runtime/ptrace_check.cc

```diff
@@ -27,6 +27,7 @@
 #include <sys/sysctl.h>
 #include <sys/types.h>
 
+#include <mach/machine.h>
```

Comment:
Should this be conditionally imported and used? @chinmaygarde might have a better idea of whether this is available on all our target platforms.

Reply:
This is fine. `TRACING_CHECKS_NECESSARY` is unset on all other platforms.